workspace_tools/targets.py

```diff
@@ -566,6 +566,23 @@ def __init__(self):
                 "template": [os.path.join(os.path.dirname(__file__), 'export', 'uvision_microlib.uvproj.tmpl')],
             }
         }
+
+class ELEKTOR_COCORICO(LPCTarget):
+    def __init__(self):
+        LPCTarget.__init__(self)
+        self.core = "Cortex-M0+"
+        self.extra_labels = ['NXP', 'LPC81X']
+        self.supported_toolchains = ["uARM", "IAR", "GCC_ARM"]
+        self.default_toolchain = "uARM"
+        self.is_disk_virtual = True
+        # ??? do I need a detect code? self.detect_code = ["1050"]
+        self.progen = {
+            "target":"CoCo-ri-Co!",  # all characters acceptable?
```

Comment:
Thank you for your comments. Here are the quick answers:

- OK, I will switch to lowercase only.

- Yes, this target provides an mbed cmsis-dap interface. So do I need the self.detect code? If so, this code or another? If another, where to get it?

> and please add this target to progen

What do you mean by that? What is progen? Where can I find it? How do I add a target to it?

Sorry for asking stupid questions, but I am totally in the dark here. I have read all sorts of porting documents, but maybe I missed one?